README.md

```diff
@@ -170,6 +170,12 @@ We can rewrite this using superlifter (see the [example code](https://github.com
   (with-superlifter context
     (s/enqueue! :pet-details (->FetchPet id))))
 ```
+Note that when defining a Superfetcher as above, the number of outputs is 
```

Comment:
@oliyh Added this note to the README, but did not update the example. I started down that road, but it was starting to get ugly, as I started adding a whole parallel set of functions to support both superfetcher styles side-by-side with different queries.

If you'd like me to make that change - or have a different suggestion for the example project - let me know.